README.md

```diff
@@ -182,7 +188,11 @@ wget https://raw.githubusercontent.com/wwood/CoverM/refs/heads/main/demo/genome_
 wget https://raw.githubusercontent.com/wwood/CoverM/refs/heads/main/demo/genome_8.fna
 ```
 
-Run CoverM
+Next, we can actually run CoverM.
+We use `--coupled` to specify the paired-end reads, and `--genome-fasta-files` to specify the genomes.
+We also specify the number of threads to use with `-t`, and the coverage metrics we want to calculate with `-m`.
```

Comment:
remove "coverage" just metrics 

You may have noticed that the coverage fraction -> covered fraction.

Notably, the relative abundances are very similar. 
-> In contrast, the relative abundance estimations do not change much when calculated with the full set of reads.